### PR TITLE
Remove timestamp hackery

### LINKED ISSUE #2786: Let's delete the timestamp hackery
~~Blocked on ares timestamps rollout~~

**Is your feature request related to a problem? Please describe.**
There are (afaik) two instances of hackery relating to the fact that ares is transitioning timestamps from a mixture of milliseconds and seconds to purely millis.

https://github.com/bbc/simorgh/blob/88c47b1c35d9ca6921eacca6e76a91aa488428d5/src/app/lib/analyticsUtils/index.js#L147

https://github.com/bbc/simorgh/pull/2780/files#diff-5a217b9ae93005d18abe435f36b7ee9aR61

Once ares rolls out all timestamps as milliseconds, we should evaluate if this hackery is still needed.

We should probably make sure that the timestamps in the fixture data are milliseconds (all such timestamps in ares are produced by multiplying by 1000 ie. end in 000, our fixture data should be the same). UPDATE the fixture data rather than replace

**Describe the solution you'd like**
Delete both(all) instances of the above hackery.

**Describe alternatives you've considered**
Refactor to avoid duplication of the above hackery.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #2786

**Overall change:** _Remove timestamp hackery that converts timestamps in seconds to milliseconds._

**Code changes:**

- _Remove timestamp hackery that converts timestamps in seconds to milliseconds._
- _Update seconds timestamps to milliseconds in tests._
- _Update timestamps in fixture data to milliseconds._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/StoryPromo/index.test.jsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ describe('StoryPromo Container', () => {
     });
 
     it('should render relative time if timestamp < 10 hours', () => {
-      const oneMinuteAgo = Math.floor(Date.now() / 1000) - 60;
+      const oneMinuteAgo = Math.floor(Date.now()) - 60 * 1000;
```
Comment (andrew-nowak) [inline src/app/containers/StoryPromo/index.test.jsx]: You won't need Math.floor if there's no division :)
```suggestion
      const oneMinuteAgo = Date.now() - 60 * 1000;
```

**Review by andrew-nowak — APPROVED** [review]
awesome! thanks for doing this, really nice job on updating all the fixture data files

**Review by j-pendlebury — APPROVED** [review]
Looks good! ⭐️

**Comment by staylos92** [comment]
This does not need testing by a tester

**Pull request merged**